brand/tokens.css

```
@@ -0,0 +1,41 @@
+  --uf-color-indigo-500: #5C49FF;
+  --uf-color-violet-500: #8F4BFF;
+  --uf-color-magenta-500: #D84BFF;
+  --uf-color-ink-900: #0F172A;
+  --uf-color-slate-600: #475569;
+  --uf-color-mist-50: #F8FAFC;
+  --uf-color-success-500: #34D399;
+  --uf-color-warning-500: #F59E0B;
+  --uf-color-danger-500: #EF4444;
+  --uf-font-display: "Satoshi", "Inter", ui-sans-serif, system-ui, sans-serif;
+  --uf-font-body: "Inter", ui-sans-serif, system-ui, sans-serif;
+  --uf-font-mono: "JetBrains Mono", ui-monospace, SFMono-Regular, Menlo, monospace;
```

Stylelint (17.14.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)